src/Propel/Generator/Model/Table.php

```diff
@@ -801,6 +801,16 @@ public function getChildrenColumn()
         return $this->inheritanceColumn;
     }
 
+    /**
+     * Checks whether the table uses concrete inheritance
+     *
+     * @return bool
+     */
+    public function usesConcreteInheritance(): bool
+    {
+        return ($this->inheritanceColumn !== null);
```

Comment:
There's no need for brackets here.

Reply:
I also dont think it hurts too much. A code sniffer some day could auto assert these things.

Reply:
@asaulenko I am aware that PHP does not need brackets around expressions. Do you believe this has serious negative impact?